modules/run_aln_comp_and_eval.nf

```diff
@@ -31,6 +31,99 @@ process run_seq_aln {
 	'''
 }
 
+process run_famsa {
+	tag "${fam}_famsa"
+	publishDir "${fam}/results", mode: 'copy'
+	cpus 4
```

Comment:
If I am not wrong all the process are using 4 cpus, so maybe you could declare it in the `nextflow.config` only once?